frontend/src/reducers/server.js

```diff
@@ -0,0 +1,21 @@
+
+const server = (state={isFetching : false, surveysArr : []},action) => {
+    switch(action.type){
+        case 'Start_Dev_Search':
+            return {
+                isFetching : true
+            }
+        break;
+
+        case 'End_Dev_Search':
+            return{
+                isFetching : false,
+                surveysArr : action.surveysArr
```

Comment:
Don't add such postfixes to variable/property names. Does it matter if it is array? 'surveys' in plural is enough.